testah-junit/src/main/java/org/testah/runner/httpLoad/HttpActor.java

```diff
@@ -51,10 +51,12 @@ public void onReceive(final Object message) throws Exception {
             } else if (message instanceof Throwable) {
                 results.get(hashId).add(getUnExpectedErrorResponseDto((Throwable) message));
             } else {
-                TS.log().info("Issue should of not made it here, message was " + message);
+                TS.log().info("Issue, should not have made it here, message is " + message);
 
             }
         } catch (Throwable throwable) {
+            TS.log().info("Throwable thrown in HttpActor.onReceive(): " + throwable.getMessage());
+            TS.log().info("Throwable thrown in HttpActor.onReceive()", throwable);
```

Comment:
you don't need the log on line 58 as it will include the message in line 59

Reply:
right - sometimes there is no message (or null), easier to spot if logged explicitly, but can change next commit